src/Model/Tag.php

```diff
@@ -207,22 +207,40 @@ public static function storeByHash(int $uriid, string $hash, string $name, strin
 	}
 
 	/**
-	 * Store tags and mentions from the body
+	 * Get tags and mentions from the body
 	 * 
-	 * @param integer $uriid   URI-Id
 	 * @param string  $body    Body of the post
 	 * @param string  $tags    Accepted tags
-	 * @param boolean $probing Perform a probing for contacts, adding them if needed
+	 *
+	 * @return array Tag list
 	 */
-	public static function storeFromBody(int $uriid, string $body, string $tags = null, $probing = true)
+	public static function getFromBody(string $body, string $tags = null)
```

Comment:
This looks like a perfect method for testing the regex expectation ;-)

Plz add a `tests/src/Model/TagTest.php` and extend it just from `MockedTest`.

Then you can easily add a test like `public function testGetFromBody()` and add some assertions :)

Reply:
"easily"

Reply:
If you like I can add the method and you add the assertions :)